pkg/operator/target_config_reconciler.go

```diff
@@ -242,6 +248,21 @@ func (c *TargetConfigReconciler) manageConfigMap(descheduler *deschedulerv1.Kube
 	return resourceapply.ApplyConfigMap(c.kubeClient.CoreV1(), c.eventRecorder, required)
 }
 
+// checkProfileConflicts ensures that multiple profiles aren't redeclared
+// it also checks for various inter-profile conflicts (profiles which should not be enabled simultaneously)
+func checkProfileConflicts(profiles sets.String, profileName deschedulerv1.DeschedulerProfile) error {
```

Comment:
You wanna write a unit test for it? :P

Reply:
Would be nice, but can be done in a followup.